mobile/src/main/java/com/gelakinetic/mtgfam/fragments/dialogs/CardViewDialogFragment.java

```diff
@@ -54,8 +55,13 @@
     /**
      * @return the currently viewed CardViewFragment in the CardViewPagerFragment
      */
+    @Nullable
     private CardViewFragment getCardViewFragment() {
```

Comment:
This isn't part of the changes you made, but the name of this should be `getParentCardViewFragment()` to remain constant with naming for all the other dialog classes.